tests/wallet/test_singleton.py

```diff
@@ -38,65 +38,66 @@ def p2_singleton_puzzle_hash(launcher_id: Program, launcher_puzzle_hash: bytes32
     return p2_singleton_puzzle(launcher_id, launcher_puzzle_hash).get_tree_hash()
 
 
-def test_only_odd_coins():
+def test_only_odd_coins() -> None:
     singleton_mod_hash = SINGLETON_MOD.get_tree_hash()
     # (SINGLETON_STRUCT INNER_PUZZLE lineage_proof my_amount inner_solution)
     # SINGLETON_STRUCT = (MOD_HASH . (LAUNCHER_ID . LAUNCHER_PUZZLE_HASH))
     solution = Program.to(
         [
             (singleton_mod_hash, (LAUNCHER_ID, LAUNCHER_PUZZLE_HASH)),
-            Program.to(binutils.assemble("(q (51 0xcafef00d 200))")),
+            Program.to(binutils.assemble("(q (51 0xcafef00d 200))")),  # type: ignore[no-untyped-call]
             [0xDEADBEEF, 0xCAFEF00D, 200],
             200,
             [],
         ]
     )
 
     with pytest.raises(Exception) as exception_info:
-        cost, result = SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
+        SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
     assert exception_info.value.args == ("clvm raise", "80")
 
     solution = Program.to(
         [
             (singleton_mod_hash, (LAUNCHER_ID, LAUNCHER_PUZZLE_HASH)),
-            Program.to(binutils.assemble("(q (51 0xcafef00d 201))")),
+            Program.to(binutils.assemble("(q (51 0xcafef00d 201))")),  # type: ignore[no-untyped-call]
             [0xDEADBEEF, 0xCAFED00D, 210],
             205,
             0,
         ]
     )
-    cost, result = SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
+    SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
 
 
-def test_only_one_odd_coin_created():
+def test_only_one_odd_coin_created() -> None:
     singleton_mod_hash = SINGLETON_MOD.get_tree_hash()
+    program = binutils.assemble("(q (51 0xcafef00d 203) (51 0xfadeddab 205))")  # type: ignore[no-untyped-call]
     solution = Program.to(
         [
             (singleton_mod_hash, (LAUNCHER_ID, LAUNCHER_PUZZLE_HASH)),
-            Program.to(binutils.assemble("(q (51 0xcafef00d 203) (51 0xfadeddab 205))")),
+            Program.to(program),
             [0xDEADBEEF, 0xCAFEF00D, 411],
             411,
             [],
         ]
     )
 
     with pytest.raises(Exception) as exception_info:
-        cost, result = SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
+        SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
     assert exception_info.value.args == ("clvm raise", "80")
-
+    clsp = "(q (51 0xcafef00d 203) (51 0xfadeddab 204) (51 0xdeadbeef 202))"
```

Comment:
`clsp_program = binutils.assemble("(q (51 0xcafef00d 203) (51 0xfadeddab 204) (51 0xdeadbeef 202))") ` would match the earlier line - I just found this somewhat odd that these two things looked a little different

Reply:
Thanks. I went with the other style because this produces a longer line (and flake8 is watching).